styles/main.css

```diff
@@ -101,13 +101,13 @@ Im not sure why this doesn't work?*/
     height: 50px;
     background-size:contain;
     
+    position: absolute;
     top: -10px; 
     right: -7px;
-    float:right;
-    z-index: 1;

```

Comment:
The lab asked for paperclip to be positioned absolutely relative to the mugshot element, which implies a `position: absolute` declaration rather than `float`